libs/index/odc/index/stac.py

```diff
@@ -130,8 +130,13 @@ def _geographic_to_projected(geometry, crs):
     geom = Geometry(geometry, 'EPSG:4326')
     geom = geom.to_crs(crs, resolution=math.inf)
 
+    # This is a somewhat dangerous assumption, if data is not in meters or you
+    # are working with very precise data!
+    def round_coords(c1, c2):
+        return [round(coord, 2) for coord in [c1, c2]]
```

Comment:
this will fail miserably for `crs==epsg:4326`.

These things are not that straightforward. One should really use pixel size as a yard stick for rounding.

Reply:
>One should really use pixel size as a yard stick for rounding.

That's true. I assumed that `projected` would be projected into a CRS that uses meters, but you're right that data could be in lat/lon natively... I'll change it.

Reply:
Ok, I have a fix that should be solid.